readthedocs/templates/core/project_bar_base.html

```diff
@@ -31,7 +31,7 @@ <h1>
           {% blocktrans trimmed %}
             Your project is currently disabled for abuse of the system.
             Please make sure it isn't using unreasonable amounts of resources or triggering lots of builds in a short amount of time.
-            Please <a href="https://github.com/rtfd/readthedocs.org/issues">file a ticket</a> to get your project re-enabled.
+            Please <a href="mailto:support@readthedocs.org">contact support</a> to get your project re-enabled.
```

Comment:
Probably we should use this from the setting

https://github.com/readthedocs/readthedocs.org/blob/c37dd7c9666832ecdddddc2a25a3bf4e3edc3b26/readthedocs/settings/base.py#L62-L62

Reply:
We could, but we need to do other things to make that possible. First, we need to define this setting in production, since it's not defined. It's just `None` there.

Also, we need to decide how to get the setting value from the template. We could create a custom template tag to get a particular setting or use a context processor.

Anyway, I think the change is small to jump into those decisions at the moment.

Reply:
You'll need to override it in .com then

Reply:
Yeah. Actually, using the .org email is better than pointing .com users to our issue tracker :D

Reply:
I'm in favor of using the setting!

Reply:
Yea, it should use the setting, and I'm 👍 on it.

Reply:
We've already doing some weirdness here that we should fix as well: 
https://github.com/readthedocs/readthedocs.org/blob/4f4a7b254c995168dbc071bd6aa68d57c34e5be6/readthedocs/core/views/__init__.py#L53-L58